### PR TITLE
Move CSS Token & Font loading to UDEX CDN

### PR DESCRIPTION
**Pull request by mhaack**
Fixes: #275 

Test URLs:
- Before: https://main--hlx-test--urfuwo.hlx.live/blog/2023/11/idc-marketscape-sap-a-leader-cloud-enabled-sourcing-applications
- After: https://css-cdn--hlx-test--urfuwo.hlx.live/blog/2023/11/idc-marketscape-sap-a-leader-cloud-enabled-sourcing-applications
- https://css-cdn--hlx-test--urfuwo.hlx.live/draft/mhaack/hero-demo1
- https://css-cdn--hlx-test--urfuwo.hlx.live/draft/mhaack/hero-demo2

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -242,30 +241,10 @@ main img {
...
-/* sections: 56px spacing at the bottom */
 main .section {
   padding-bottom: 56px;
 }
 
-
-/**
- * Styles for embedding video content.
- */
-.video-embed-container {
```
Comment (pahupe) [inline styles/styles.css:254]: Out of curiosity: why have these been removed? Are these now part of the UDex resources?

File: head.html [hunk 2]
```diff
@@ -10,4 +10,5 @@
 </script>
 <script src="/scripts/aem.js" type="module"></script>
 <script src="/scripts/scripts.js" type="module"></script>
+<!-- <link rel="stylesheet" href="https://cdn.udex.services.sap.com/dds/design-tokens/v0.36.0/css/sap_glow.css"/> -->
```
Comment (pahupe) [inline head.html:13]: Why would this comment be left in head.html?

**Review by pahupe — APPROVED** [review]
LGTM, I haven't found any issues on pages I tested with.

**Review by saurabh-khare — APPROVED** [review]
We will have to see later if any variables we use is not missing in the variable definitions

File: head.html (hunk 2 above)
Comment (mhaack) [reply head.html]: Good catch, removed it

File: styles/styles.css (hunk 1 above)
Comment (mhaack) [reply styles/styles.css:254]: This will be remove by https://github.com/urfuwo/hlx-test/pull/257, was letf over from some testing

**Comment by saurabh-khare** [comment]
@mhaack Do you want to merge this PR today? I want to fix fonts for cards with new vars once this is merged.

**Pull request closed without merging**